PullRequestCreated/PullRequestCreated.ts

```diff
@@ -16,6 +22,25 @@ class PullRequestCreated extends OctokitAction {
   private shouldCreateIssue(pr: PullRequest): boolean {
     return pr != null && !JIRA_ISSUE_PATTERN.test(pr.title);
   }
+
+  private async newIssueTypeAndParent(pr: PullRequest): Promise<Issue> {
```

Comment:
This, or something that is aligned with the new class name
```suggestion
  private async newIssueParameters(pr: PullRequest): Promise<...fixme...> {
```